### ISSUE TITLE
Support debug on SpyBeans

### ISSUE THREAD

**Issue by raulvaldoleirospowerdot** [redhat-developer/vscode-java#3653]
Can't debug a class annotated with SpyBean annotation, this makes hard to debug testes because I need to comment the annotation and all the verify statements.

I uploaded a project as sample, if you uncomment the SpyBean annotation you will be able to debug it.

[sample-project.zip](https://github.com/redhat-developer/vscode-java/files/15324540/sample-project.zip)

Thanks

**Comment by snjeza** [comment]
I can't reproduce the issue.
![spybean](https://github.com/redhat-developer/vscode-java/assets/388609/4ad4343c-5448-4363-95ac-a76f9e5a680a)

**Comment by raulvaldoleirospowerdot** [comment]
Sorry, I forgot to mention the class.

Can you try to add the breakpoint on line 9 of ComponentSample class please?

**Comment by snjeza** [comment]
I can reproduce the issue in Eclipse too.

**Comment by raulvaldoleirospowerdot** [comment]
That means the fix should be done first in eclipse and then raise the version on vscode java?
Just to understand the process 😀
Thanks

**Comment by rgrunber** [comment]
Yes, it's a possibility that the issue is in a component coming from JDT (and not JDT-LS). @snjeza is there anything in the client/server logs to indicate what the failure might be ?

**Comment by e-dong** [comment]
See workaround from this stackoverflow post
https://stackoverflow.com/questions/61775881/eclipse-mockito-debugging-doesnt-stop-on-breakpoints-in-spied-beans

**Comment by e-dong** [comment]
Has anyone raised the issue in Eclipse?

**Comment by jacknight** [comment]
> See workaround from this stackoverflow post https://stackoverflow.com/questions/61775881/eclipse-mockito-debugging-doesnt-stop-on-breakpoints-in-spied-beans

Tihs worked for me in vscode, thanks.
